lib/ice_cube/time_util.rb

```diff
@@ -198,6 +200,36 @@ def self.which_occurrence_in_month(time, wday)
       [nth_occurrence_of_weekday, this_weekday_in_month_count]
     end
 
+    # Use Activesupport CoreExt functions to manipulate time
+    def self.start_of_month time
+      time.beginning_of_month
+    end
+
+    # Use Activesupport CoreExt functions to manipulate time
+    def self.end_of_month time
+      time.end_of_month
+    end
+
+    # Use Activesupport CoreExt functions to manipulate time
+    def self.start_of_year time
+      time.beginning_of_year
+    end
+
+    # Use Activesupport CoreExt functions to manipulate time
+    def self.end_of_year time
+      time.end_of_year
+    end
+
+    # Use Activesupport CoreExt functions to manipulate time
+    def self.previous_month time
+      time - 1.month
+    end
+
+    # Use Activesupport CoreExt functions to manipulate time
```

Comment:
I don't understand the point of adding those class methods to `TimeUtils`.
Wouldn't it be easier to just call `time.whatever` instead of `TimeUtils.whatever(time)` wherever we need those methods?